packages/logs/src/domain/logger.ts

```diff
@@ -61,79 +49,105 @@ export class Logger {
     }
   }
 
-  @monitored
   log(message: string, messageContext?: object, status: StatusType = StatusType.info, error?: Error) {
-    let errorContext: RawLoggerLogsEvent['error']
+    // note: generating the handling stack is intentionally duplicated in each of the logger method to save a frame in the stack
+    let handlingStack: string | undefined
 
-    if (error !== undefined && error !== null) {
-      const stackTrace = error instanceof Error ? computeStackTrace(error) : undefined
-      const rawError = computeRawError({
-        stackTrace,
-        originalError: error,
-        nonErrorPrefix: NonErrorPrefix.PROVIDED,
-        source: ErrorSource.LOGGER,
-        handling: ErrorHandling.HANDLED,
-        startClocks: clocksNow(),
-      })
-
-      errorContext = {
-        stack: rawError.stack,
-        kind: rawError.type,
-        message: rawError.message,
-        causes: rawError.causes,
-      }
+    if (isAuthorized(status, HandlerType.http, this)) {
+      handlingStack = createHandlingStack()
     }
 
-    const sanitizedMessageContext = sanitize(messageContext) as Context
-
-    const context = errorContext
-      ? (combine({ error: errorContext }, sanitizedMessageContext) as Context)
-      : sanitizedMessageContext
-
-    this.handleLogStrategy(
-      {
-        message: sanitize(message)!,
-        context,
-        status,
-      },
-      this
-    )
+    this.logImplementation(message, messageContext, status, error, handlingStack)
   }
 
   ok(message: string, messageContext?: object, error?: Error) {
-    this.log(message, messageContext, StatusType.ok, error)
+    let handlingStack: string | undefined
+
+    if (isAuthorized(StatusType.ok, HandlerType.http, this)) {
+      handlingStack = createHandlingStack()
+    }
+
+    this.logImplementation(message, messageContext, StatusType.ok, error, handlingStack)
   }
```

Comment:
> always create the handlingstack

Indeed, this would save ~660 Bytes as per my calculations. However generating a stack trace could be expensive, especially if used a lot.

Another solution is to skip 3 frame instead of 2 for logs. Customers can always increase the stack size if needed.

wdyt?

Reply:
Apply the prototype trick we talked during the daily. +240B vs +909B before 🎉 
Thank you @BenoitZugmeyer for the suggestion